src/gui_common/ActionButton.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Godot;
+
+/// <summary>
+///   A hotkey representing and visualizing an input for in-game actions.
+/// </summary>
+public class ActionButton : Button
+{
+    private Panel highlight;
+    private TextureRect iconRect;
```

Comment:
no abbreviations

Reply:
I've accepted usage of rect because it's literally in the name of the Godot class, I'll add it to: https://github.com/Revolutionary-Games/Thrive/issues/2392